src/SDL12_compat.c

```diff
@@ -3258,16 +3275,6 @@ SetPalette12ForMasks(SDL12_Surface *surface12, const Uint32 Rmask, const Uint32
             color->a = 255;
         }
 
-        format20 = surface12->surface20->format;
```

Comment:
The `format20` local var can be removed now in this procedure

Reply:
Removed, thanks!